isis/tests/Fixtures.cpp

```diff
@@ -21,6 +23,14 @@ namespace Isis {
     testCube = new Cube();
     testCube->fromIsd(tempDir.path() + "/default.cub", label, isd, "rw");
 
+    LineManager line(*testCube);
```

Comment:
Decided to use a LineManager because it was easier to follow along with. Could have used a Brick